codingtest_ajw/20일차/실패율.js

```diff
@@ -0,0 +1,20 @@
+function solution(N, stages) {
+    let answer = [];
```

Comment:
```suggestion
    const answer = [];
```
배열이나 객체 같은 경우 배열, 객체 그 차제의 값이 바뀌는게 아니라면, 안에있는 값을 참조해서 변경하는 것이기 때문에 ```const```로 선언하는게 좋습니다.

원시값과, 참조값에 대해서 공부해 보시면 좋을 듯 싶습니다.
[참고사이트](https://hanamon.kr/javascript-%EB%B3%80%EC%88%98%EC%9D%98-%ED%83%80%EC%9E%85-%EC%9B%90%EC%8B%9C%ED%98%95%EA%B3%BC-%EC%B0%B8%EC%A1%B0%ED%98%95/)